bin/functional-tests/conftest.py

```diff
@@ -21,7 +21,27 @@ def create_kube_client(in_cluster=False):
         config.load_kube_config()
     return client.CoreV1Api()
 
-# Create a test fixture for the prometheus pod
+@pytest.fixture(scope='session')
+def nginx(request):
+    """ This is the host fixture for testinfra. To read more, please see
+    the testinfra documentation:
+    https://testinfra.readthedocs.io/en/latest/examples.html#test-docker-images
+    """
+    namespace = os.environ.get('NAMESPACE')
+    release_name = os.environ.get('RELEASE_NAME')
+    if not namespace:
+        print("NAMESPACE env var is not present, using 'astronomer' namespace")
+        namespace = 'astronomer'
+    if not release_name:
+        print("RELEASE_NAME env var is not present, assuming 'astronomer' is the release name")
+        release_name = 'astronomer'
+    kube = create_kube_client()
+    pods = kube.list_namespaced_pod(namespace, label_selector=f"component=ingress-controller")
```

Comment:
```suggestion
    pods = kube.list_namespaced_pod(namespace, label_selector="component=ingress-controller")
```

No need for f-string.